policy/release/tasks.rego

```diff
@@ -246,9 +246,7 @@ deny contains result if {
 # custom:
 #   short_name: unsupported
 #   failure_msg: >-
-#     Task %q is used by pipeline task %q is or will be unsupported as of %s.
-#   solution: >-
-#     Upgrade to a newer version of the Task.
```

Comment:
Not sure how important it is for the solution to be in `solution` rather than in `failure_msg`. I can look into making the solution template-able, but I would need some help with that

Reply:
I think this is fine.